src/app/action-playground/action-playground.component.html

```diff
@@ -66,3 +66,13 @@
 
 <strong>Show only one value (Uses the language of the user if logged in. If not, uses the browser language)</strong>
 <p>{{labels | dspStringifyStringLiteral}}</p>
+
+<hr>
+<p>StringLiteral input example</p>
+<dsp-string-literal-input [placeholder]="'List label'"> </dsp-string-literal-input>
```

Comment:
Can you add the example with the output here and display the values as console.log? 
e.g.

```typescript
label: StringLiteral[];
...
handleData(data: StringLiteral[]) {
  this.label = data;
}

submitData() {
  console.log('submit string literal', this.label);
}
```

```html
<dsp-string-literal-input [placeholder]="'List label'" (dataChanged)="handleData($event)" (enter)="submitData()""></dsp-string-literal-input>

Reply:
added in ae0538c